### PR TITLE
fix null variables

### PR DESCRIPTION
**Pull request by cdiop**

### CODE REVIEW

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java [hunk 1]
```diff
@@ -136,7 +136,7 @@
 
     private final int widthWindows = 600;
```
Comment (sonarqube-activeeon) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java]: MINOR Make this final field static too. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java [hunk 2]
```diff
@@ -136,7 +136,7 @@
...
 
-    private final int heightWindows = 620;
+    private final int heightWindows = 600;
```
Comment (sonarqube-activeeon) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java]: MINOR Make this final field static too. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 25 issues
* ![MAJOR][MAJOR] 15 major
* ![MINOR][MINOR] 10 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SubmitWindow.java#L564](https://github.com/ow2-proactive/scheduling-portal/blob/af7afaa512bf4dbdfbcc2402f4d1a19ffc05cf6c/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L564): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L575](https://github.com/ow2-proactive/scheduling-portal/blob/af7afaa512bf4dbdfbcc2402f4d1a19ffc05cf6c/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L575): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L583](https://github.com/ow2-proactive/scheduling-portal/blob/af7afaa512bf4dbdfbcc2402f4d1a19ffc05cf6c/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L583): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L679](https://github.com/ow2-proactive/scheduling-portal/blob/af7afaa512bf4dbdfbcc2402f4d1a19ffc05cf6c/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L679): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L792](https://github.com/ow2-proactive/scheduling-portal/blob/af7afaa512bf4dbdfbcc2402f4d1a19ffc05cf6c/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L792): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L910](https://github.com/ow2-proactive/scheduling-portal/blob/af7afaa512bf4dbdfbcc2402f4d1a19ffc05cf6c/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L910): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L1078](https://github.com/ow2-proactive/scheduling-portal/blob/af7afaa512bf4dbdfbcc2402f4d1a19ffc05cf6c/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L1078): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L1095](https://github.com/ow2-proactive/scheduling-portal/blob/af7afaa512bf4dbdfbcc2402f4d1a19ffc05cf6c/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L1095): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L1112](https://github.com/ow2-proactive/scheduling-portal/blob/af7afaa512bf4dbdfbcc2402f4d1a19ffc05cf6c/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L1112): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L1195](https://github.com/ow2-proactive/scheduling-portal/blob/af7afaa512bf4dbdfbcc2402f4d1a19ffc05cf6c/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L1195): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java [hunk 3]
```diff
@@ -392,9 +392,26 @@ private void fillVarsPart(String jobDescriptor) {
     }
 
     private Widget prepareWorlflowInformationWidget() {
-        return new HTML("<center> <img src=" + icon + " height=40px width=40px> <br> <b>Documentation :</b> <a href=" +
-                        documentation + " target=_blank>" + documentation + " </a><br> <b> Bucket name :</b> " +
-                        bucketNameGI + " </center>");
+
+        String widgetIcon = "Please add an icon to the workflow <br>";
```
Comment (chloegugli) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java]: Is it possible to add the icon from this interface? If not, I think showing nothing might be a better solution than this message

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java [hunk 4]
```diff
@@ -392,9 +392,26 @@ private void fillVarsPart(String jobDescriptor) {
...
+        String widgetDocumentation = "Please add a documentation link to the workflow <br>";
```
Comment (chloegugli) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java]: Same comment, is it possible to add documentation from this interface? If not, I think showing "No documentation" or a message like that might be a better solution. When you say "Please add a documentation link to the workflow ", the user might think the documentation is mandatory to submit the workflow. Is it mandatory?

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java [hunk 5]
```diff
@@ -392,9 +392,26 @@ private void fillVarsPart(String jobDescriptor) {
...
+        String widgetBucketName = "Please add a bucket name to the workflow";
```
Comment (chloegugli) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java]: Same comment, is it possible to add bucket name from this interface?

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java (hunk 3 above)
Comment (cdiop) [reply scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java]: I put "No icon"

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java (hunk 4 above)
Comment (cdiop) [reply scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java]: I put "No documentation"

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java (hunk 5 above)
Comment (cdiop) [reply scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java]: I put "No bucketName"

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java [hunk 6]
```diff
@@ -392,9 +392,26 @@ private void fillVarsPart(String jobDescriptor) {
     }
 
     private Widget prepareWorlflowInformationWidget() {
-        return new HTML("<center> <img src=" + icon + " height=40px width=40px> <br> <b>Documentation :</b> <a href=" +
-                        documentation + " target=_blank>" + documentation + " </a><br> <b> Bucket name :</b> " +
-                        bucketNameGI + " </center>");
+
+        String widgetIcon = "No icone <br>";
```
Comment (MaelAudren) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java]: icone => icon

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java (hunk 1 above)
Comment (sonarqube-activeeon) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java]: MINOR Make this final field static too. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java (hunk 2 above)
Comment (sonarqube-activeeon) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java]: MINOR Make this final field static too. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 25 issues
* ![MAJOR][MAJOR] 15 major
* ![MINOR][MINOR] 10 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SubmitWindow.java#L581](https://github.com/ow2-proactive/scheduling-portal/blob/422c82ca74f854408b2f4de432f6ca98769381f4/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L581): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L592](https://github.com/ow2-proactive/scheduling-portal/blob/422c82ca74f854408b2f4de432f6ca98769381f4/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L592): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L600](https://github.com/ow2-proactive/scheduling-portal/blob/422c82ca74f854408b2f4de432f6ca98769381f4/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L600): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L696](https://github.com/ow2-proactive/scheduling-portal/blob/422c82ca74f854408b2f4de432f6ca98769381f4/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L696): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L809](https://github.com/ow2-proactive/scheduling-portal/blob/422c82ca74f854408b2f4de432f6ca98769381f4/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L809): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L927](https://github.com/ow2-proactive/scheduling-portal/blob/422c82ca74f854408b2f4de432f6ca98769381f4/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L927): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L1094](https://github.com/ow2-proactive/scheduling-portal/blob/422c82ca74f854408b2f4de432f6ca98769381f4/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L1094): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L1111](https://github.com/ow2-proactive/scheduling-portal/blob/422c82ca74f854408b2f4de432f6ca98769381f4/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L1111): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L1128](https://github.com/ow2-proactive/scheduling-portal/blob/422c82ca74f854408b2f4de432f6ca98769381f4/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L1128): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [SubmitWindow.java#L1211](https://github.com/ow2-proactive/scheduling-portal/blob/422c82ca74f854408b2f4de432f6ca98769381f4/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java#L1211): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**